components/hostobjects/org.jaggeryjs.hostobjects.file/src/main/java/org/jaggeryjs/hostobjects/file/FileHostObject.java

```diff
@@ -369,4 +369,178 @@ public static Object jsFunction_listFiles(Context cx, Scriptable thisObj, Object
         return cx.newArray(thisObj, fhol.toArray());
     }
 
+    /**
+     * To unzip a zip file
+     *
+     * @param cx      Context
+     * @param thisObj FileHostObject to be unzipped
+     * @param args    Path to unzip the zip file
+     * @param funObj  Function Object
+     * @throws ScriptException
+     */
+    public static boolean jsFunction_unZip(Context cx, Scriptable thisObj, Object[] args, Function funObj)
+            throws ScriptException, IOException {
+        String functionName = "unZip";
+        int argsCount = args.length;
```

Comment:
I think when there are no arguments, it will come as empty array. So we do not need to check null. Similar way is followed in other methods in this class as well.